prow/apis/prowjobs/v1/types.go

```diff
@@ -1046,6 +1046,13 @@ func (r Refs) String() string {
 	return strings.Join(rs, ",")
 }
 
+func (r Refs) OrgRepoString() string {
+	if len(r.Repo) != 0 {
```

Comment:
Nit: Doing empty string comparisons by comparing against an empty string (`r.Repo == ""`) rather than checking the length makes this more readable, as it is immediately obvious that the variable type is a string

Reply:
fixed!